scripts/new-runtime-e2e-tests/configs/storage.json

```diff
@@ -1,22 +1,18 @@
 {
-  "Sudo": {
-    "Key": "23Z6s3egDsnShpy9jnTe8gGjnW5bm5iBKhrsGtZMjtSP79D6"
-  },
   "System": {
     "Account": [
       [
-        ["23Z6s3egDsnShpy9jnTe8gGjnW5bm5iBKhrsGtZMjtSP79D6"],
+        ["5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY"],
         {
-          "providers": 1,
```

Comment:
It needs the `providers` here

Reply:
oh this is an obsolete file, storage override is in `{karura,acala}.yml`